tensorbay/geometry/box.py

```diff
@@ -139,7 +113,59 @@ def iou(box1: "Box2D", box2: "Box2D") -> float:
         return intersect / union
 
     @classmethod
-    def loads(cls: Type[_T], contents: Dict[str, float]) -> _T:
+    def from_xywh(cls: Type[_B2], x: float, y: float, width: float, height: float) -> _B2:
+        """Create a :class:`Box2D` instance from the top-left vertex and the width and the height.
+
+        Arguments:
+            x: X coordinate of the top left vertex of the box.
+            y: Y coordinate of the top left vertex of the box.
+            width: Length of the box along the x axis.
+            height: Length of the box along the y axis.
+
+        Returns:
+            The created :class:`Box2D` instance.
+
+        """
+        obj: _B2 = object.__new__(cls)
+        if width <= 0 or height <= 0:
+            obj._data = (0.0,) * Box2D._LENGTH  # pylint: disable=protected-access
+            return obj
+
+        obj._data = (x, y, x + width, y + height)  # pylint: disable=protected-access
+        return obj
+
+    @classmethod
+    def from_center_xywh(cls: Type[_B2], x: float, y: float, width: float, height: float) -> _B2:
+        """Create a :class:`Box2D` instance from the center point and the width and the height.
+
+        Arguments:
+            x: X coordinate of the box center.
+            y: Y coordinate of the box center.
+            width: Length of the box along the x axis.
+            height: Length of the box along the y axis.
+
+        Returns:
+            The created :class:`Box2D` instance.
+
+        """
+        obj: _B2 = object.__new__(cls)
+        if width <= 0 or height <= 0:
+            obj._data = (0.0,) * Box2D._LENGTH  # pylint: disable=protected-access
+            return obj
+
+        half_width = width / 2
```

Comment:
python `/` will transfer `int` to `float` which will make the coordinate type unexpected